lib/trino-orc/src/test/java/io/trino/orc/OrcTester.java

```diff
@@ -1351,6 +1377,25 @@ private static Type rowType(Type... fieldTypes)
         return TESTING_TYPE_MANAGER.getParameterizedType(StandardTypes.ROW, typeSignatureParameters.build());
     }
 
+    private static boolean isTime(Type type)
+    {
+        if (type.equals(TIME_MICROS)) {
+            return true;
+        }
+        if (type instanceof ArrayType arrayType) {
+            return isTime(arrayType.getElementType());
```

Comment:
`isTime( "array(time)" )` feels wrong

rename `isTime` method to `containsTimeMicros`